common/locales/en/contrib.json

```diff
@@ -54,8 +54,9 @@
     "playerTiers": "Player Tiers",
     "tier": "Tier",
     "visitHeroes": "Visit the Hall of Heroes (contributors and backers)",
-    "conLearn": "Learn more about contributor rewards",
+    "conLearn": "<a href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank'>Learn more about contributor rewards</a>",
```

Comment:
Do you think I should separate the uris and the text?
On Wed, Aug 10, 2016 at 10:56 AM Kaitlin Hipkin <notifications@github.com>
wrote:

> In common/locales/en/contrib.json
> <https://github.com/HabitRPG/habitrpg/pull/7036#discussion_r74273538>:
>
> > @@ -54,8 +54,9 @@
> >      "playerTiers": "Player Tiers",
> >      "tier": "Tier",
> >      "visitHeroes": "Visit the Hall of Heroes (contributors and backers)",
> > -    "conLearn": "Learn more about contributor rewards",
> > +    "conLearn": "<a href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank'>Learn more about contributor rewards</a>",
>
> @MathWhiz <https://github.com/MathWhiz> Here's an example of what I mean.
> This conLearn string is actually a link, whereas the conLearnLink string
> below is just a URL, not an actual link.
>
> Again, it's just a suggestion to clarify the intended usage.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7036/files/965ec666377117066360b0564e4caa525f9c5ffb#r74273538>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_82uzChiXfsdV71lp3f8NJyAQS3APks5qefSxgaJpZM4ICNL2>
> .
>